### PR TITLE
Fix flaky tests

### PR DESCRIPTION
**Pull request by submarcos**
## Checklist

- [ ] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/contribute/contributing.html)
- [ ] My code respects the Definition of done available in the [Development section of the documentation](https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields)
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes
- [ ] I added an entry in the changelog file
- [ ] My commits are all using prefix convention (emoji + tag name) and references associated issues
- [ ] I added a label to the PR corresponding to the perimeter of my contribution
- [ ] The title of my PR mentionned the issue associated

### CODE REVIEW

File: geotrek/infrastructure/tests/test_parsers.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def setUpTestData(cls):
 
     @mock.patch('requests.get')
     @mock.patch('requests.head')
-    @override_settings(MODELTRANSLATION_DEFAULT_LANGUAGE="fr")
+    @override_settings(MODELTRANSLATION_DEFAULT_LANGUAGE="fr", LANGUAGE_COE="fr")
```
Comment (Chatewgne) [inline geotrek/infrastructure/tests/test_parsers.py:33]:
```suggestion
    @override_settings(MODELTRANSLATION_DEFAULT_LANGUAGE="fr", LANGUAGE_CODE="fr")
```

**Review by Chatewgne — CHANGES_REQUESTED** [review]

File: geotrek/common/middleware.py [hunk 2]
```diff
@@ -10,9 +10,17 @@ def get_language_from_path(path):
     regex_match = language_code_prefix_re.match(path)
     if not regex_match:
         return None
-    lang_code = regex_match.group(1)
+    return regex_match.group(1)
+
+
+def get_language_from_request(request):
+    request.GET.get('lang')
```
[the remark this answers is not in the document]
Comment (submarcos) [reply geotrek/common/middleware.py]: good point. fixed

File: geotrek/cirkwi/tests/test_views.py [hunk 3]
```diff
@@ -43,6 +42,7 @@ def setUp(self):
         self.poi.save()
 
     def test_export_circuits(self):
+        self.maxDiff = None
```
[the remark this answers is not in the document]
Comment (submarcos) [reply geotrek/cirkwi/tests/test_views.py]: fixed

File: geotrek/cirkwi/tests/test_views.py [hunk 4]
```diff
@@ -155,6 +155,7 @@ def test_export_pois_with_attachments(self):
         )
 
     def test_export_circuits_with_attachments(self):
+        self.maxDiff = None
```
[the remark this answers is not in the document]
Comment (submarcos) [reply geotrek/cirkwi/tests/test_views.py]: fixed

File: geotrek/infrastructure/tests/test_parsers.py (hunk 1 above)
Comment (submarcos) [reply geotrek/infrastructure/tests/test_parsers.py]: fixed

**Pull request merged**